src/lib/libwebaudio.js

```diff
@@ -183,7 +183,7 @@ let LibraryWebAudio = {
 
     // TODO: In MINIMAL_RUNTIME builds, read this file off of a preloaded Blob,
     // and/or embed from a string like with WASM_WORKERS==2 mode.
-    audioWorklet.addModule('{{{ TARGET_BASENAME }}}.aw.js').then(() => {
+    audioWorklet.addModule(locateFile('{{{ TARGET_BASENAME }}}.aw.js')).then(() => {
```

Comment:
How about
```
audioWorklet.addModule(
#if MINIMAL_RUNTIME
  '{{{ TARGET_BASENAME }}}.aw.js'
#else
  locateFile('{{{ TARGET_BASENAME }}}.aw.js')
#endif
).then(() => {
```

The `locateFile` function does not currently exist in `MINIMAL_RUNTIME`.

Reply:
I think we also need the `new URL(...)` pattern here as well for ES6 bundlers.

Essentially copy the pattern from regular ww.js workers.

Reply:
@sbc100 At this point, maybe this should be a parseTools macro for arbitrary resources?

Reply:
If you can think of way to express that that SGTM

Reply:
Since original PR author might not be around, I posted a PR https://github.com/emscripten-core/emscripten/pull/24044 to gate the MINIMAL_RUNTIME part, and add a test.

parseTools macro and ES6 bundlers are good ideas that can be added in later PRs.